mysql-test/suite/innodb/r/xtradb_compressed_columns_alter_table.result

```diff
@@ -1403,3 +1403,44 @@ modified_dictionary_references_cleaned
 
 DROP COMPRESSION_DICTIONARY dict1;
 DROP COMPRESSION_DICTIONARY dict2;
+#
+# Bug PS-1108 : Changing a column from uncompressed to compressed for JSON crashes the server
+#
+CREATE TABLE t1 (json_variables JSON);
+INSERT INTO t1 VALUES('{"name":"json test"}');
+ALTER TABLE t1 MODIFY json_variables JSON COLUMN_FORMAT COMPRESSED;
```

Comment:
One last request. Please add the reverse scenario as well.
COMPRESSED JSON to normal JSON.